sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/SignInResult.java

```diff
@@ -5,6 +5,8 @@
 
 package com.microsoft.appcenter.identity;
 
+import com.microsoft.appcenter.UserInformation;
```

Comment:
Maybe I didn't explain correctly in previous comments.
Just revert this change.